### DISCUSSION TITLE
象鼻岩斷掉要改叫什麼?

### DISCUSSION THREAD

**Discussion by Sherry9106** [1101-GS4719-C/Python-Community-GS4719-C#1799] [General]
最近很多臉書好友發布把象鼻岩黏回去的活動，我覺得超好笑的，但我自己是沒親自去看過象鼻岩啦，也沒有體會過它的壯觀，比較有體驗的是野柳的女王頭，看新聞說近幾年也會面臨斷頭的危機。現在象鼻岩斷掉了，大家覺得名字要改叫什麼呀 ? 感覺會有很多有趣的取名><
https://www.msn.com/zh-tw/news/national/%E8%B1%A1%E9%BC%BB%E5%B2%A9%E6%96%B7%E9%BC%BB%E6%89%BE%E5%88%B0%E4%BA%86-%E7%A9%BA%E6%8B%8D%E7%9B%B4%E6%93%8A%E7%95%AB%E9%9D%A2%E6%9B%9D%E5%85%89/ar-AA1lFsU9?cvid=09f6e2c8fcf944e6a1aa81ea2a2fc2c3&ei=28

**Comment by 111426027** [comment]
我是覺得沒看到好可惜，但要叫什麼還真的沒想過。

**Comment by v0ouo** [comment]
最近真的有不少好笑的活動阿，像是辦在我們學校的也有一堆，比如最近的丟松果測時速大賽，很常看到有人半夜在丟松果，有人甚至把測速儀砸出一個洞，另外還有一些活動像是oloo環校比賽，也都蠻有趣的。

**Reply by v0ouo** [reply]
結果這禮拜丟松果測時速大賽就被取消了，果然辦活動還是要小心。看到有人丟到手脫臼及把測速儀玻璃打破，真的要小心。

**Comment by 12Jesse24** [comment]
象岩感覺不錯聽(因為鼻沒了)，喔不超冷的。對不起在這麼冷的天還上來討論象鼻岩斷掉要叫做什麼><

**Comment by Patsy17** [comment]
斷鼻岩～
不過對於這件事情真的感到很遺憾，一個自然環境就這樣沒有了

**Comment by mrweixd** [comment]
建議改名大岩壁，立志成為大學生觀光景點
